gluster.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I have a problem with name of this playbook file: name `gluster.yml` suggests that this is concerned with gluster only, but it also installs tendrl components for gluster storage. For this reason, could we rename this to `tendrl_gluster_server.yml` (or something similar)?